Sources/StreamChatUI/CommonViews/Attachments/AttachmentsPreviewVC.swift

```diff
@@ -115,25 +120,6 @@ open class AttachmentsPreviewVC: _ViewController, ComponentsProvider {
     }
     
     open func setupVerticalStackView() {
-        // Disable scroll when not needed
-        scrollView.isScrollEnabled = content.count > maxNumberOfVerticalItems
-
-        // If the content is bigger than the max vertical items and the scroll view height
-        // constraint is not yet created, append to the vertical constraint and activate it.
-        if content.count > maxNumberOfVerticalItems {
-            if scrollViewHeightConstraint == nil {
-                scrollViewHeightConstraint = scrollView.heightAnchor.pin(
-                    lessThanOrEqualToConstant: scrollView.frame.size.height
-                )
-                scrollViewHeightConstraint?.isActive = true
```

Comment:
The `setupVerticialStackView()` is always called on the update content, so why don't we always update the constant value of the scrollViewHeightConstraint based on the current content? I'm really not convinced with the current solution since besides causing a breaking change, it is not very flexible. At least we need to extract the hard coded "54" to an open variable so that is easily customizable, otherwise, the customer needs to add additional constraints for this. Also, Visually we can also break the UI for some customers if they have already attachments views that have a bigger height, and I know some customers who have this, so with the current solution not only we will break the API but also the UI, so we need to come up with something better, even if that involves some workarounds.